src/core/agent/services/identifierService.ts

```diff
@@ -303,6 +315,144 @@ class IdentifierService extends AgentService {
       );
     }
   }
+
+  async createMultisig(
+    ourIdentifier: string,
+    otherIdentifierContacts: GenericRecord[],
```

Comment:
Not sure why this changed to `GenericRecord` - the UI shouldn't need to deal with these types when calling `createMultisig`

Reply:
We fetch other contacts from our metadata records and then the creator select them to create the multisig. I changed it as your suggestion in this comment https://github.com/cardano-foundation/cf-identity-wallet/pull/309#discussion_r1436940982

Reply:
This was my mistake, sorry! I forgot that returned `GenericRecord[]`.

In the UI, we already have `ConnectionShortDetails[]` as our list of connections, can we filter this to KERI-only to present to the user, and use this to select?

We would need to add the OOBI URL to `ConnectionShortDetails` (I think the `label` is the URL for KERI right now, but this might change so I would add a dedicated `oobi` field)

What do you think?

Reply:
Will need to add more checks, but I will update it.